### DISCUSSION TITLE
Clickable targetMarker

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#2123] [Q&A] [answered]
### Introduction

Hello everyone,

I have a link with a targetMarget. I would like to make it clickable. I managed to make the link more clickable by expanding the clickable field with the path "detectionAreaCharge" but I didn't manage to add a clickable area on the targetMarker. 

Thanks
![code](https://user-images.githubusercontent.com/83480447/230425068-b4c84682-70a1-4409-813c-ae6ab30aaa3b.png)
![Capture d’écran 2023-04-06 171847](https://user-images.githubusercontent.com/83480447/230425093-8d159e1c-d4d3-4ece-9cd0-72b46bd0bdc1.png)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
If you use `targetMarker`, `sourceMarker` or `vertexMarker`, JointJS will create them using the [SVGMarker](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/marker). And as per specification, the SVG markers do not capture events. See [StackOverflow answer](https://stackoverflow.com/a/38222510).

The arrowhead needs to be created using the `<path>` (or any [basic shape](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Basic_Shapes)).

Here's an [example](https://jsfiddle.net/kumilingus/L3p2qwam/).

**Reply by malomiquel** [reply]
Thank you for your help this work well !
